eng/pipelines/jobs/credscan_matrix.yml

```diff
@@ -0,0 +1,66 @@
+parameters:
+  - name: Matrix
+    type: string
+  - name: TestTimeoutInMinutes
+    type: number
+    default: 60
+  - name: CloudConfig
+    type: object
+    default: {}
+  - name: UsePlatformContainer
+    type: boolean
+    default: false
+  - name: DependsOn
+    type: object
+    default: []
+jobs:
+  - job:
+    displayName: 'CredScan'
+    condition: |
+      and(
+        succeededOrFailed(),
+        ne(${{ parameters.Matrix }}, '{}'),
+        ne(${{ parameters.Matrix}}, '{"":{"Tag":null}}')
+      )
+    timeoutInMinutes: ${{ parameters.TestTimeoutInMinutes }}
+
+    strategy:
+      matrix: $[ ${{ parameters.Matrix }} ]
```

Comment:
Is it necessary to really do this in parallel? I suspect it isn't work the extra work and copying of files to do this. We probably can get away with just running it all serial.

Reply:
We could, but `steps` are generated at compile time, and I much prefer this view from a diagnosability standpoint.

As we discussed in 1:1, I'll remove the matrix generation in favor of a simple one. That'll get rid of the extra supporting files, but keep the main advantage I was looking for here.